homeassistant/components/renson/strings.json

```
@@ -151,5 +151,100 @@
+        "level1": "[%key:component::renson::entity::sensor::ventilation_level::state::level1%]",
+        "level2": "[%key:component::renson::entity::sensor::ventilation_level::state::level2%]",
+        "level3": "[%key:component::renson::entity::sensor::ventilation_level::state::level3%]",
+        "level4": "[%key:component::renson::entity::sensor::ventilation_level::state::level4%]",
+        "breeze": "[%key:component::renson::entity::sensor::ventilation_level::state::breeze%]",
+        "holiday": "Holiday"
+      }
+    }
+  },
+  "services": {
+    "set_timer_level": {
+      "name": "Set timer",
+      "description": "Set the ventilation timer",
+      "fields": {
+        "timer_level": {
+          "name": "Level",
+          "description": "Level setting"
+        },
+        "time": {
+          "name": "Time",
+          "description": "Time of the timer (0 will disable the timer)"
+        }
+      }
+    },
+    "set_breeze": {
+      "name": "Set breeze",
+      "description": "Set the breeze function of the ventilation system",
+      "fields": {
+        "breeze_level": {
+          "name": "Level",
+          "description": "Ventilation level when breeze function is activated"
+        },
+        "temperature": {
+          "name": "Temperature",
+          "description": "Temperature when the breeze function should be activated"
+        },
+        "activate": {
+          "name": "Activate",
+          "description": "Activate or disable the breeze feature"
+        }
+      }
+    },
+    "set_day_night_time": {
+      "name": "Set day and night time",
+      "description": "Set the day and night time of the system",
+      "fields": {
+        "day": {
+          "name": "Start day",
+          "description": "Start time of the day"
+        },
+        "night": {
+          "name": "Start night",
+          "description": "Start time of the night"
+        }
+      }
+    },
+    "set_pollution_settings": {
+      "name": "Set pollution settings",
+      "description": "Set all the pollution settings of the ventilation system",
+      "fields": {
+        "day_pollution_level": {
+          "name": "Day pollution Level",
+          "description": "Ventilation level when pollution is detected in the day"
+        },
+        "night_pollution_level": {
+          "name": "Night pollution Level",
+          "description": "Ventilation level when pollution is detected in the night"
+        },
+        "humidity_control": {
+          "name": "Enable humidity control",
+          "description": "Activate or disable the humidity control"
+        },
+        "airquality_control": {
+          "name": "Enable air quality control",
+          "description": "Activate or disable the air quality control"
+        },
+        "co2_control": {
+          "name": "Enable CO2 control",
+          "description": "Activate or disable the CO2 control"
+        },
+        "co2_threshold": {
+          "name": "co2 threshold",
```

```suggestion
          "name": "CO2 threshold",
```

Please be consistent about "CO2"